package.json

```diff
@@ -7,21 +7,26 @@
     "node": ">= 4"
   },
   "scripts": {
-    "build": "babel src --out-dir ./dist/src",
+    "build:source": "babel src --out-dir ./dist/src",
+    "build:test": "babel test --out-dir ./dist/test --copy-files",
```

Comment:
nit: I usually prefer gnu style options `--out-dir=./dist/test`

Reply:
Does not work for `mocha`'s `--opts=./mocha.opts` unfortunately. I'll leave it as it is for consistency.

Reply:
Oh mocha 🤦‍♂️  OK

Reply:
😏 the last commits don't contain --opts anymore 😄 but let it be